deployments/puppet/README.md

```diff
@@ -66,13 +66,14 @@ This class accepts the following parameters:
 | `splunk_memory_total_mib`            | Total memory in MIB to allocate to the collector; automatically calculates the ballast size.  The `SPLUNK_MEMORY_TOTAL_MIB` environment variable will be set with this value for the collector service.                                                                                                                                                                                                                                                                                                                                                                                                                                      | `512`                                                                                                                         |
-| `splunk_ballast_size_mib`            | Set the ballast size for the collector explicitly instead of the value calculated from the `$splunk_memory_total_mib` parameter.  This should be set to 1/3 to 1/2 of configured memory.  The `SPLUNK_BALLAST_SIZE_MIB` environment variable will be set with this value for the collector service.                                                                                                                                                                                                                                                                                                                                          | None                                                                                                                          |
+|**DEPRECATED** `splunk_ballast_size_mib`            | Set the ballast size for the collector explicitly instead of the value calculated from the `$splunk_memory_total_mib` parameter.  This should be set to 1/3 to 1/2 of configured memory.  The `SPLUNK_BALLAST_SIZE_MIB` environment variable will be set with this value for the collector service.                                                                                                                                                                                                                                                                                                                                          | None                                                                                                                          |
```

Comment:
```suggestion
|**DEPRECATED** `splunk_ballast_size_mib`            | Set the ballast size for the collector explicitly instead of the value calculated from the `$splunk_memory_total_mib` parameter.  This should be set to 1/3 to 1/2 of configured memory.  The `SPLUNK_BALLAST_SIZE_MIB` environment variable will be set with this value for the collector service. Only applicable if `collector_version` is < 0.97.0.                                                                                                                                                                                                                                                                                                                                         | None                                                                                                                          |
```